docs/src/configuration.md

```diff
@@ -42,39 +42,39 @@ Let's look at an example window definition:
+Here, we are defining a window named `example`, which we then define a set of properties for. Additionally, we set the content of the window to be the text `"example content"`.
 
 You can now open your first window by running `eww open example`! Glorious!
 
 ### `defwindow`-properties
 
 |   Property | Description                                                  |
 | ---------: | ------------------------------------------------------------ |
-|  `monitor` | which monitor this window should be displayed on.            |
+|  `monitor` | Which monitor this window should be displayed on.            |
 | `geometry` | Geometry of the window.  |
 
 
-**Geometry-properties**
+### `geometry`-properties
 
 | Property          | Description |
 | -----------------:| ------------------------------------------------------------ |
-| `x`, `y`          | Position of the window. Values may be provided in `px` or `%`. Will be relative to `anchor` |
+|          `x`, `y` | Position of the window. Values may be provided in `px` or `%`. Will be relative to `anchor`. |
 | `width`, `height` | Width and height of the window. Values may be provided in `px` or `%`. |
-| `anchor`          | Anchor-point of the window. Either `center` or combinations of `top`, `center`, `bottom` and `left`, `center`, `right` |
-
+|          `anchor` | Anchor-point of the window. Either `center` or combinations of `top`, `center`, `bottom` and `left`, `center`, `right`. |
 
+<br/>
 Depending on if you are using X11 or wayland, some additional properties exist:
 
-#### x11
+### x11
 
 |     Property | Description                                                  |
 | -----------: | ------------------------------------------------------------ |
 |   `stacking` | Where the window should appear in the stack. Possible values: `fg`, `bg`. |
-|  `wm-ignore` | Whether the windowmanager should ignore this window. This is useful for dashboard-style widgets that don't need to interact with other windows at all. Note that this makes some of the other properties not have any effect. Either `true` or `false` |
-|    `reserve` | Specify how the window-manager should make space for your window. This is useful for bars, which should not overlap any other windows. |
+|  `wm-ignore` | Whether the window manager should ignore this window. This is useful for dashboard-style widgets that don't need to interact with other windows at all. Note that this makes some of the other properties not have any effect. Either `true` or `false`. |
+|    `reserve` | Specify how the window manager should make space for your window. This is useful for bars, which should not overlap any other windows. |
 | `windowtype` | Specify what type of window this is. This will be used by your window manager to determine how it should handle your window. Possible values: `normal`, `dock`, `toolbar`, `dialog`. Default: `dock` if `reserve` is specified, `normal` otherwise. |
 
-#### wayland
+### wayland
```

Comment:
The heading changes here don't really make sense. x11 and wayland are indeed subheadings of the `defwindow properties`, and if anything, the geometry properties should be, too - although given that they aren't distinct in the same way that x11 and wayland are, keeping them as simply **fat** seems reasonable